misc/testerina/modules/testerina-core/src/main/ballerina/execute.bal

```diff
@@ -56,6 +57,7 @@ public function startSuite() returns int {
             }
             executeAfterSuiteFunctions();
             reportGenerators.forEach(reportGen => reportGen(reportData));
+            println(string `${"\n"}${"\t"}${"\t"}Test execution time : ${executionTime}ms`);
```

Comment:
```suggestion
            println(string `${"\n"}${"\t"}${"\t"}Test execution time: ${executionTime}ms`);
```

Reply:
We can do that. But, sometimes we get them as a small decimal number(I mean, 3 ms is printed as 0.003s). It may be not accurate for small test cases. And, we use milliseconds while dumping the build time as well.